modules.d/35network-legacy/module-setup.sh

```diff
@@ -25,10 +25,12 @@ installkernel() {
 install() {
     local _arch
 
-    #Adding default link
-    if dracut_module_included "systemd"; then
+    # Add default link if there is no persistent network device naming
+    if [ ! -e /etc/udev/rules.d/70-persistent-net.rules ] &&\
+         dracut_module_included "systemd"; then
+
         inst_multiple -o "${systemdutildir}/network/99-default.link"
-        [[ $hostonly ]] && inst_multiple -H -o "${systemdsystemconfdir}/network/*.link"
+        [[ $hostonly ]] && inst_multiple -H -o "$ {systemdutilconfdir}/network/*.link"
```

Comment:
There is a blank space between `$` and `{` in `$ {systemdutilconfdir}`

Reply:
Oops, ok, fixed.